docs/integrations/switchboard.md

```diff
@@ -0,0 +1,331 @@
+---
+title: Switchboard
+head:
+  - - meta
+    - name: title
+      content: Solana Cookbook | Using Switchboard to create Onchain data feeds
+  - - meta
+    - name: og:title
+      content: Solana Cookbook | Using Switchboard to create Onchain data feeds
+  - - meta
+    - name: description
+      content: Switchboard allows builders to unlock the power of Solana by creating high performance data feeds from any API.
+  - - meta
+    - name: og:description
+      content: Switchboard allows builders to unlock the power of Solana by creating high performance data feeds from any API.
+  - - meta
+    - name: og:image
+      content: https://solanacookbook.com/cookbook-sharing-card.png
+  - - meta
+    - name: og:image:alt
+      content: Solana splash card
+  - - meta
+    - name: twitter:card
+      content: summary
+  - - meta
+    - name: twitter:site
+      content: "@solanacookbook"
+  - - meta
+    - name: twitter:image
+      content: "https://solanacookbook.com/cookbook-sharing-card.png"
+  - - meta
+    - name: robots
+      content: index,follow,noodp
+  - - meta
+    - name: googlebot
+      content: index,follow
+---
+
+# Switchboard
+
+Switchboard is a protocol that allows builders to unlock the power of Solana by creating high performance data feeds from any API. Switchboard can be used for various applications like getting current price of an asset for lending, getting the current weather or flight data and determining current NBA match status.
```

Comment:
Switchboard is an Oracle protocol that enables developers to source data on-chain for a variety of use cases such as price feeds, NFT floor prices, sport statistics, or even verifiable randomness. In a general sense, Switchboard is an off-chain resource developers can invoke to bridge high integrity data on-chain and power the next generation of web3 and DeFi.